app/scripts/models/account.js

```diff
@@ -414,10 +416,13 @@ define(function (require, exports, module) {
         }
       })
       .then((updatedSessionData) => {
+        this._metrics.logEvent('core.signin.success');
```

Comment:
@shane-tomlinson I wonder if emitting the `signin.success` metric before the user gets to the [ready.js](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/views/complete_sign_up.js#L209) view might be a better place? That way it doesn't matter if they close the tab or not at polling view, it would still get emitted. They would not get redirected to the view if any verification errors so we wouldn't emit duplicate metrics.